hummingbot/connector/exchange/ftx/ftx_constants.py

```diff
@@ -0,0 +1,71 @@
+import sys
+
+from hummingbot.core.api_throttler.data_types import LinkedLimitWeightPair, RateLimit
+
+HBOT_BROKER_ID = "hummingbot1"
+MAX_ORDER_ID_LEN = None
```

Comment:
Would be good if we can validate this.

Reply:
What needs validation @CrimsonJacket? The value configured in `HBOT_BROKER_ID` or the value configured in `MAX_ORDER_ID_LEN`?
The first one is the value requested by the foundation (check one of the first comments in https://github.com/hummingbot/hummingbot/pull/5438)
For the second one, I could not find a max number of characters for the clientId. I looked for in in the API documentation and also in the example implementation (https://github.com/ftexchange/ftx) but neither of them mentions a limit